src/software/pipeline/main_panoramaCompositing.cpp

```diff
@@ -595,6 +595,11 @@ int aliceVision_main(int argc, char** argv)
         return EXIT_FAILURE;
     }
 
+    // set maxThreads
+    HardwareContext hwc = cmdline.getHardwareContext();
+    hwc.setUserCoresLimit(maxThreads);
+    omp_set_num_threads(hwc.getMaxThreads());
```

Comment:
Maybe a function setupMaxThreads()?

`cmdline.getHardwareContext().setupMaxThreads();`

Reply:
Should we use openmp at this level ?